ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml

```diff
@@ -6,7 +6,19 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-vexxhost
+    mount_path: /var/run/cluster-secrets-openstack-vexxhost
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-moc
+    mount_path: /var/run/cluster-secrets-openstack-moc
```

Comment:
The two credential secrets, cluster-secrets-openstack-vexxhost and cluster-secrets-openstack-moc can be removed if we put all cloud credentials in one single secret and use OS_CLOUD to select.

Reply:
done